mkt/extensions/validation.py

```diff
@@ -70,10 +79,13 @@ def __init__(self, file_obj=None):
         self.file_obj = file_obj
         self.zipfile = None
 
-    def error(self, error_key):
+    def error(self, error_key, error_parameter=None):
+        message = self.errors[error_key]
+        if error_parameter is not None:
+            message = self.errors[error_key] % error_parameter
         raise ParseError(detail={
             'key': error_key,
-            'message': self.errors[error_key],
+            'message': message,
         })
```

Comment:
I was thinking something this simple:

    def error(self, key, *args):
        raise ParseError(detail={
            'key': key,
            'message': self.errors[key] % args,
            'params': args,
        }) 

Then errors can have any number of substitution tokens (often none; `'string' % ()` is valid), so long as the message for the key expects as many as is passed.